src/main/java/org/opentripplanner/middleware/otp/response/LocalizedAlert.java

```diff
@@ -46,31 +57,19 @@ public String getAlertDescriptionForHtml() {
     public int hashCode() {
         // Exclude effectiveEndDate from the hash code for cases where a given alert is "extended",
         // e.g. incidents that take longer to resolve than initially planned.
-        return Objects.hash(alertHeaderText, alertDescriptionText, alertUrl, effectiveStartDate);
+        return Objects.hash(getAlertHeaderText(), getAlertDescriptionText(), alertUrl, effectiveStartDate);
     }
 
     public boolean equals(Object o) {
         if (!(o instanceof LocalizedAlert)) {
             return false;
         }
         LocalizedAlert ao = (LocalizedAlert) o;
-        if (alertDescriptionText == null) {
-            if (ao.alertDescriptionText != null) {
-                return false;
-            }
-        } else {
-            if (!alertDescriptionText.equals(ao.alertDescriptionText)) {
-                return false;
-            }
+        if (!getAlertDescriptionText().equals(ao.getAlertDescriptionText())) {
```

Comment:
I think these two if statement can be combined.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/215/commits/4c14f95ab15888c47bf285cd5a9f688d49f3f94b.